expertise/models/bm25/bm25.py

```diff
@@ -49,6 +51,9 @@ def set_archives_dataset(self, archives_dataset):
     def set_submissions_dataset(self, submissions_dataset):
         self.submissions_dataset = submissions_dataset
 
+    def _is_valid_field(self, obj, field):
+        return field in obj and len(obj.get(field)) > 0
```

Comment:
@carlosmondra how about using this instead:
```suggestion
        return field in obj and obj.get(field)
```
This avoids 2 errors:
1. If the field does not exist in obj, the `len(None)` operation would error out.
2. If the field exists but is not a type that supports len() operation, it would error out.